doc/user_manual/sampler.tex

```diff
@@ -1695,6 +1695,10 @@ \subsubsection{Custom Sampling strategy}
       \textbf{ProbabilityWeight} quantities are not found, the samples are assumed to come from a MonteCarlo (from a  statistical
       post-processing prospective).
   \end{itemize}
+ \item \xmlNode{index}, \xmlDesc{comma-separated integer, optional parameter} indexes to use from the
```

Comment:
is it a list of integers or a single integer? because in the InputData specifications, you are using a IntegerType (i.e. single integer and not a comma separated list of integers)